lib/mayaUsd/resources/scripts/mayaUsdClearRefsOrPayloadsOptions.py

```diff
@@ -0,0 +1,188 @@
+#
+# Copyright 2023 Autodesk
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import maya.cmds as cmds
+import maya.mel as mel
+
+from mayaUsdLibRegisterStrings import getMayaUsdLibString
+import mayaUsdOptions
+
+from functools import partial
+
+kAllRefsCheckboxName = "allRefsCheckbox"
+kAllPayloadsCheckboxName = "allPayloadsCheckbox"
+
+_itemName = None
+
+def showClearRefsOrPayloadsOptions(itemName):
+    """
+    Shows the clear references or payloads options dialog.
+    """
+    global _itemName
+    _itemName = itemName
+    title = getMayaUsdLibString("kClearRefsOrPayloadsOptionsTitle")
+    userChoice = mel.eval('''layoutDialog -title "%s" -parent "MayaWindow" -uiScript createClearRefsOrPayloadsOptionsDialog;''' % title)
+
+    results = [userChoice]
+
+    options = getClearRefsOrPayloadsOptionsDict()
+    for optionName in ['references', 'payloads']:
+        if optionName in options and options[optionName]:
+            results.append(optionName)
+
+    return results
+
+
+def _createClearRefsOrPayloadsOptionsDialog():
+    """
+    Creates the clear references or payloads dialog UI.
+    """
+
+    # Same spacing as Maya option boxes.
+    spacer = 16
+    edge   = 12
```

Comment:
Where did you get these values from? They are double the ones defined in mayaUsdMergeToUSDOptions.py

Reply:
With the other values, the buttons looked too cramped compared to the design in the story.

Reply:
I re-tested it with the values from the other file and it does not look too bad, so I will use them as they are the Maya defaults.